src/components/Tutorial/__tests__/Tutorial.ts

```diff
@@ -0,0 +1,296 @@
+  beforeEach(() => createTestApp({ tutorial: true }))
+  afterEach(cleanupTestEventHandlers)
+  afterAll(cleanupTestApp)
+  it('step start - tell about context menu', async () => {
+    await user.click(screen.getByText('Learn more'))
+
+    ensureStep(TUTORIAL2_STEP_START)
+    expect(screen.getByText(`If the same thought appears in more than one place`, { exact: false })).toBeInTheDocument()
+    expect(
+      screen.getByText(`shows a small number to the right of the thought, for example`, { exact: false }),
+    ).toBeInTheDocument()
+  })
+
+  it('step choose - gives 3 choices of what project to proceed with', async () => {
+    await user.click(screen.getByText('Next'))
+    ensureStep(TUTORIAL2_STEP_CHOOSE)
+    expect(screen.getByText(/For this tutorial, choose what kind of content you want to create/)).toBeInTheDocument()
+    expect(screen.getByText('To-Do List')).toBeInTheDocument()
+    expect(screen.getByText('Journal Theme')).toBeInTheDocument()
+    expect(screen.getByText('Book/Podcast Notes')).toBeInTheDocument()
+  })
+
+  it('step context 1 parent - we create "Home" thought with children', async () => {
+    await user.click(screen.getAllByText('To-Do List').at(-1)!)
+    expect(screen.getByText(/Excellent choice. Now create a new thought with the text “Home”/)).toBeInTheDocument()
+    ensureStep(TUTORIAL2_STEP_CONTEXT1_PARENT)
+
+    // we create a `Home` thought
+    await user.keyboard('{Enter}')
+    await user.type(lastThought(), 'Home')
+    await act(vi.runOnlyPendingTimersAsync)
+    ensureStep(TUTORIAL2_STEP_CONTEXT1)
+
+    // Home -> To Do
+    expect(screen.getByText(/Let's say that you want to make a list of things/)).toBeInTheDocument()
+    expect(screen.getByText(/Add a thought with the text "To Do"/)).toBeInTheDocument()
+    await user.keyboard('{Control>}{Enter}{/Control}')
+    await user.type(lastThought(), 'To Do')
+    await act(vi.runOnlyPendingTimersAsync)
+    ensureStep(TUTORIAL2_STEP_CONTEXT1_SUBTHOUGHT)
+
+    // Home -> To Do -> or to not
+    expect(screen.getByText(/Now add a thought to “To Do”/)).toBeInTheDocument()
+    await user.keyboard('{Control>}{Enter}{/Control}')
+    await user.type(lastThought(), 'or to not')
+    await act(vi.runOnlyPendingTimersAsync)
+
+    expect(exportContext(store.getState(), [HOME_TOKEN], 'text/plain')).toBe(
+      `- __ROOT__
+  - Home
+    - To Do
+      - or to not`,
+    )
+    expect(screen.getByText(/Nice work!/)).toBeInTheDocument()
+  })
+
+  it('step context 2 - we create "Work" thought with children', async () => {
+    await user.click(screen.getByText('Next'))
+    ensureStep(TUTORIAL2_STEP_CONTEXT2_PARENT)
+    expect(screen.getByText(/Now we are going to create a different "To Do" list./)).toBeInTheDocument()
+
+    // we created a new thought on 3rd level, so we shift-tab our way back to root
+    await user.keyboard('{Enter}')
+    await user.keyboard('{Shift>}{Tab}{/Shift}')
+    await user.keyboard('{Shift>}{Tab}{/Shift}')
```

Comment:
I don't mind this, but it might be more natural to select a thought at the root and create the new thought from there with one `enter` (and not use `shift+tab`).

Thoughts, @raineorshine?

Reply:
Yes, I agree. Shift + Tab activates the Outdent command. It would be better to stick to simple navigation and New Thought commands for completing the tutorial.

Reply:
totally makes sense, thanks
now it's
```jsx
await user.click(screen.getByText('Home'))
await user.keyboard('{Enter}')
```